pkgs/development/compilers/xa/dxa.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchurl
+, installShellFiles
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dxa";
+  version = "0.1.4";
+
+  src = fetchurl {
+    url = "https://www.floodgap.com/retrotech/xa/dists/${pname}-${version}.tar.gz";
+    hash = "sha256-C0rgwK51Ij9EZCm9GeiVnWIkEkse0d60ok8G9hm2a5U=";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  dontConfigure = true;
+
+  postPatch = ''
+    sed -i \
+        -e 's/^CC = gcc/CC = cc/' \
+        Makefile
```

Comment:
Can we replace this with substituteInPlace?

Reply:
YEP